### DISCUSSION TITLE
Login/password not recognised on android app

### DISCUSSION THREAD

**Discussion by mikerjensen** [immich-app/immich#9014] [Q&A] [answered]
### The bug

App was working fine last week. On the weekend i upgraded the server and android app to the latest version and now the app cannot log in (reports 'Error logging you in, check server URL, email and password') but i can log in no problem with the same credentials via web browser.  Log info pasted below. I have Immich set up as a local instance, and oddly, since the weekend Immich server cannot be found by the Android app at the URL  i was using before (http://photos.local) but can be found at http://photos.lan in the app - via the web browser either url works fine.

------
Logout failed for [lankamike@gmail.com](mailto:lankamike@gmail.com)

ApiException 400: Socket operation failed: POST /auth/logout (Inner exception: Failed host lookup: 'photos.local')

#0      IOClient.send (package:http/src/io_client.dart:94)
<asynchronous suspension>
#1      BaseClient._sendUnstreamed (package:http/src/base_client.dart:93)
<asynchronous suspension>
#2      ApiClient.invokeAPI (package:openapi/api_client.dart:96)
<asynchronous suspension>
#3      AuthenticationApi.logout (package:openapi/api/authentication_api.dart:140)
<asynchronous suspension>
#4      AuthenticationNotifier.logout.<anonymous closure> (package:immich_mobile/modules/login/providers/authentication.provider.dart:106)
<asynchronous suspension>

#0      ApiClient.invokeAPI (package:openapi/api_client.dart:104)
<asynchronous suspension>
#1      AuthenticationApi.logout (package:openapi/api/authentication_api.dart:140)
<asynchronous suspension>
#2      AuthenticationNotifier.logout.<anonymous closure> (package:immich_mobile/modules/login/providers/authentication.provider.dart:106)
<asynchronous suspension>

### The OS that Immich Server is running on

Debian Bullseye

### Version of Immich Server

latest ver - 1.102.0~ynh1

### Version of Immich Mobile App

v1.102.2

### Platform with the issue

- [X] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
none
```

### Your .env content

```Shell
not sure where to find this
```

### Reproduction steps

```bash
1. load http://photos.lan  on the android app
2. Responds with login and passwords field for immich
3. After entering the correct details (known to be correct due to logging in on web), the app says error logging in.
...
```

### Relevant log output

```shell
Logout failed for [lankamike@gmail.com](mailto:lankamike@gmail.com)

ApiException 400: Socket operation failed: POST /auth/logout (Inner exception: Failed host lookup: 'photos.local')

#0      IOClient.send (package:http/src/io_client.dart:94)
<asynchronous suspension>
#1      BaseClient._sendUnstreamed (package:http/src/base_client.dart:93)
<asynchronous suspension>
#2      ApiClient.invokeAPI (package:openapi/api_client.dart:96)
<asynchronous suspension>
#3      AuthenticationApi.logout (package:openapi/api/authentication_api.dart:140)
<asynchronous suspension>
#4      AuthenticationNotifier.logout.<anonymous closure> (package:immich_mobile/modules/login/providers/authentication.provider.dart:106)
<asynchronous suspension>


#0      ApiClient.invokeAPI (package:openapi/api_client.dart:104)
<asynchronous suspension>
#1      AuthenticationApi.logout (package:openapi/api/authentication_api.dart:140)
<asynchronous suspension>
#2      AuthenticationNotifier.logout.<anonymous closure> (package:immich_mobile/modules/login/providers/authentication.provider.dart:106)
<asynchronous suspension>
```

**Comment by alextran1502** [comment]
Can you try updating your server to v1.102.3?

**Comment by alextran1502** [comment, marked as answer]
can you also use direct IP address please?

**Reply by mikerjensen** [reply]
Thanks for the suggestions, the yunohost update to 1.102.3 came through this morning, so i just updated, verified that the new version is running by logging in with the web browser, which worked fine, however same problem occurs - cannot log in with the android app, either when trying to access via direct IP or when using the domain name (photos.lan - photos.local not recognised).

**Reply by alextran1502** [reply]
Can you make sure that your phone is on the same local network? Can you use the mobile browser to see if you can access the instance? What is your phone model?

**Reply by mikerjensen** [reply]
Thanks, for the suggestion. My phone is a Nord 1+, however i think it was the local suffix .lan that i had used - I decided to reinstall immich on photos.local rather than photos.lan and that seemed to solve the problem.
